crates/ln-dlc-node/src/tests/just_in_time_channel.rs

```diff
@@ -22,17 +22,17 @@ async fn just_in_time_channel() {
     payer.keep_connected(coordinator.info).await.unwrap();
     payee.keep_connected(coordinator.info).await.unwrap();
 
-    // Fund the on-chain wallets of the nodes who will open a channel
-    payer.fund(Amount::from_sat(100_000)).await.unwrap();
     coordinator.fund(Amount::from_sat(100_000)).await.unwrap();
 
-    let coordinator_outbound_liquidity_sat = 25_000;
-    let payer_inbound_liquidity_sat = 25_000;
+    let payer_outbound_liquidity_sat = 25_000;
+    let coordinator_outbound_liquidity_sat =
+        min_outbound_liquidity_channel_creator(&payer, payer_outbound_liquidity_sat);
+
     let payer_coordinator_channel_details = coordinator
         .open_channel(
-            &payer.info,
+            &payer,
             coordinator_outbound_liquidity_sat,
-            payer_inbound_liquidity_sat,
+            payer_outbound_liquidity_sat,
```

Comment:
🙃 depends on the point of view 😜

Reply:
The thing is the coordinator's outbound liquidity is the payer's inbound liquidity, so we can't use both variables together.